_component_design/context-drawer.md

```diff
@@ -0,0 +1,539 @@
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Context drawer easing animation"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/content-areas/context-drawer/context-drawer-animation-easing.png" width="675"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Duration
+
+Duration is the amount of time the drawer takes to animate from off-stage to on-stage, and from on-stage to off-stage.
+
+Each drawer size has a unique animation duration so that the visual weight and movement looks natural.
+
+The drawer slides out twice as fast as it slides in because the closing animation requires less user focus than does the user’s next task.
+
+**Animating into view**
+
+To calculate animation duration for sliding the drawer into view, use the following formula:
+
+*SlideInDuration = (DrawerWidthPX * 0.8)/1000*
+
+For example: STILL NEED AN EXAMPLE FROM DREW
+
+**Animating out of view**
+
+The animation duration to slide out is calculated as follows:
+
+*SlideOutDuration = SlideInDuration/2*
+
+For example: STILL NEED AN EXAMPLE FROM DREW
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+<video controls loop autoplay>
+  <source src="{{site.url}}/assets/images/components/content-areas/context-drawer/context-drawer-animation-duration.webm" type="video/webm" codecs="vp8, vorbis">
+  Your browser does not support HTML5 video.
+</video>
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## States
+
+A context drawer can exist in the following states: **loading**, **failure to load**, **form submission**, **validation error**, **application error**, and **success**.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Loading
+
+In scenarios where the content of the drawer must be loaded, the drawer begins the loading process immediately after being triggered by the  user.
+
+Show the loading spinner centered horizontally and vertically within the body of the drawer.
+
+Hide the footer until the contents loads.
+
+When the content has loaded, hide the spinner and show the footer if there is one.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Context drawer loading state"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/content-areas/context-drawer/context-drawer-states-loading.png" width="436"/>
```

Comment:
I noticed a few small inconsistencies in the drawer images.  I'm going to send you new images for a few of these.

Once that's done, I think we're good to go.

Reply:
Cool, just let me know when those are available!

Reply:
visuals replaced, done.